selfdrive/car/mazda/carstate.py

```diff
@@ -65,12 +65,15 @@ def update(self, cp, cp_cam):
     # Either due to low speed or hands off
     lkas_blocked = cp.vl["STEER_RATE"]["LKAS_BLOCK"] == 1
 
-    # LKAS is enabled at 52kph going up and disabled at 45kph going down
-    # wait for LKAS_BLOCK signal to clear when going up since it lags behind the speed sometimes
-    if speed_kph > LKAS_LIMITS.ENABLE_SPEED and not lkas_blocked:
-      self.lkas_allowed_speed = True
-    elif speed_kph < LKAS_LIMITS.DISABLE_SPEED:
-      self.lkas_allowed_speed = False
+    carFP = self.CP.carFingerprint
+
+    if carFP not in (CAR.CX5_2022):
```

Comment:
set carParams.minSteerSpeed in interface, then check that here

Reply:
Set minSteetSpeed appropriately per supported car you mean?

Reply:
yeah

Reply:
Can we leave this as-is for now ?  I need to do some clean up to prepare support for newer gen Mazdas

Reply:
I'd like to do this one properly. I can do it if you want

Reply:
Sure, otherwise, I can do it late tonight. Thanks @adeebshihadeh

Reply:
Gave you head start @adeebshihadeh!